src/appkit.cs

```diff
@@ -884,6 +880,29 @@ interface NSApplication : NSAccessibilityElementProtocol, NSUserInterfaceValidat
 		bool ProtectedDataAvailable { [Bind ("isProtectedDataAvailable")] get; }
 	}
 
+	[NoMacCatalyst]
+	[Category]
+	[BaseType (typeof(NSApplication))]
+	interface NSApplication_NSServicesMenu {
+		[Export ("registerServicesMenuSendTypes:returnTypes:"), EventArgs ("NSApplicationRegister")]
+		void RegisterServicesMenu (string [] sendTypes, string [] returnTypes);
+	}
+
+	[NoMacCatalyst]
+	[Category]
+	[BaseType (typeof(NSApplication))]
+	interface NSApplication_NSStandardAboutPanel {
+		[Export ("orderFrontStandardAboutPanel:")]
+		void OrderFrontStandardAboutPanel ([NullAllowed] NSObject sender);
+
+		[Export ("orderFrontStandardAboutPanelWithOptions:")]
+#if XAMCORE_5_0
```

Comment:
XAMCORE_5_0, when XAMCORE deprecated you... `</joke>`

Reply:
aww I had my review opened in another computer, you beat me to it 😂